src/category_theory/groupoid/free_groupoid.lean

```diff
@@ -158,21 +157,21 @@ quotient.lift _
       symmetry,
       apply groupoid.comp_inv, })
 
-lemma lift_spec (φ : prefunctor V V') : (of V).comp (lift φ).to_prefunctor = φ :=
+lemma lift_spec (φ : V ⥤q V') : of V ⋙q (lift φ).to_prefunctor = φ :=
 begin
   rw [of_eq, prefunctor.comp_assoc, prefunctor.comp_assoc, functor.to_prefunctor_comp],
   dsimp [lift],
   rw [quotient.lift_spec, paths.lift_spec, quiver.symmetrify.lift_spec],
 end
 
-lemma lift_unique (φ : prefunctor V V') (Φ : free_groupoid V ⥤ V')
-  (hΦ : (of V).comp Φ.to_prefunctor = φ) : Φ = (lift φ) :=
+lemma lift_unique (φ : V ⥤q V') (Φ : free_groupoid V ⥤ V')
+  (hΦ : of V ⋙q Φ.to_prefunctor = φ) : Φ = lift φ :=
 begin
   apply quotient.lift_unique,
   apply paths.lift_unique,
-  apply quiver.symmetrify.lift_unique,
+  apply @quiver.symmetrify.lift_unique _ _ _ _ _ _ _ _ _,
```

Comment:
Why this regression?

Reply:
We made the property that a prefunctor preserves arrow reversal into a class.
In some previous commit, I had a `haveI : …` line before the apply, but that got changed to the `@ _ ` version.